docs/docs/10.4-test-utils.md

```diff
@@ -12,6 +12,8 @@ next: clone-with-props.html
 var ReactTestUtils = require('react-addons-test-utils');
 ```
 
+>Note: Airbnb has a released a new testing utility called [Enzyme](http://airbnb.io/enzyme/), which makes it easy to assert, manipulate, and traverse your React Components' output.  If you're deciding on a unit testing library, it's worth checking out: [http://airbnb.io/enzyme/](http://airbnb.io/enzyme/)
```

Comment:
A couple things.
1. I don't think this needs to be a "Note:" but if you're going to do it like that please format it correctly (this will render the entire text bold)
2. Probably don't need to link to it twice
3. Are there other testing libraries worth linking to?
4. "... released a new ..." will get dated. Want to phrase it in a way that's a bit more timeless since we probably won't change this text for 3 years ;)